File: CS/E3534/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraLayout;
using DevExpress.Utils.Win;

namespace E3534
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void searchLookUpEdit1_Popup(object sender, EventArgs e)
        {
            Control window = (sender as IPopupControl).PopupWindow;
            //Direct reference to the LayoutControl. It could be changed in future versions.
            //LayoutControl lc = (sender as IPopupControl).PopupWindow.Controls[2].Controls[0] as LayoutControl;
            LayoutControl lc = SearchForLayout(window);
            LayoutControlGroup lcgroup = lc.Items[0] as LayoutControlGroup;
            lcgroup.AppearanceGroup.BackColor = Color.CadetBlue;
        }

        LayoutControl SearchForLayout (Control container)
        {
            foreach (Control control in container.Controls)
            {
                if (control is LayoutControl)
                    return control as LayoutControl;
                LayoutControl lc = SearchForLayout(control);
                if (lc != null)
                    return lc;
            }
            return null;
        }
    }
}

File: VB/E3534/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraLayout
Imports DevExpress.Utils.Win

Namespace E3534
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub searchLookUpEdit1_Popup(ByVal sender As Object, ByVal e As EventArgs) Handles searchLookUpEdit1.Popup
			Dim window As Control = (TryCast(sender, IPopupControl)).PopupWindow
			'Direct reference to the LayoutControl. It could be changed in future versions.
			'LayoutControl lc = (sender as IPopupControl).PopupWindow.Controls[2].Controls[0] as LayoutControl;
			Dim lc As LayoutControl = SearchForLayout(window)
			Dim lcgroup As LayoutControlGroup = TryCast(lc.Items(0), LayoutControlGroup)
			lcgroup.AppearanceGroup.BackColor = Color.CadetBlue
		End Sub

		Private Function SearchForLayout(ByVal container As Control) As LayoutControl
			For Each control As Control In container.Controls
				If TypeOf control Is LayoutControl Then
					Return TryCast(control, LayoutControl)
				End If
				Dim lc As LayoutControl = SearchForLayout(control)
				If lc IsNot Nothing Then
					Return lc
				End If
			Next control
			Return Nothing
		End Function
	End Class
End Namespace